src/lib/gui/mat_to_gl.h

```diff
@@ -31,9 +29,8 @@ namespace mat_to_gl {
 			default:
 				;
 		}
-		glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA8, mat.cols, mat.rows, 0, format, GL_UNSIGNED_BYTE, mat.data);
+		glTexImage2D(GL_TEXTURE_2D, 0, GL_RGB8, mat.cols, mat.rows, 0, format, GL_UNSIGNED_BYTE, mat.data);
```

Comment:
`glTexImage2D` seems finicky (at least in webGL, where we care), so we're focusing on the 24-bit RGB case that our inputs will theoretically match...